components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -50,6 +51,54 @@ - (instancetype)initWithFrame:(CGRect)frame {
   return self;
 }
 
+#pragma mark - Property Getter
+
+- (NSArray<MDCChipView *> *)chipViews {
+  return [self.mutableChipViews copy];
+}
+
+#pragma mark - Public API
+
+- (void)appendChipWithText:(NSString *)text {
+  MDCChipView *chipView = [[MDCChipView alloc] init];
+  chipView.titleLabel.text = text;
+  chipView.translatesAutoresizingMaskIntoConstraints = NO;
+
+  [self appendChipView:chipView];
+}
+
+- (void)appendChipView:(MDCChipView *)chipView {
+  if ([self.chipTextFieldDelegate respondsToSelector:@selector(chipField:shouldAddChip:)]) {
+    NSInteger indexToAppend = self.mutableChipViews.count;
```

Comment:
Why is there an index, if we're always adding to the end?  Also, if we're always adding to the end, we might as well call it "shouldAppendChip" and remove the index argument.